### ISSUE TITLE
CAMPD Home Page-Sprint 5.1 508 Issue 

### ISSUE THREAD

**Issue by JanellC** [US-EPA-CAMD/PI-1#1817]
## Required Fix 

- [x] RF-1
- [x] RF-2

 

## Context 
See audit report: https://app.zenhub.com/files/287570343/04aa9407-04e4-4189-b6fc-884e8fbe1d06/download

| Issue # | Issue Title | Issue Description |
|---|---|---|
| RF-1 | Color   Contrast is Insufficient  | Color   contrast ratio between text and background is less than 4.5:1. This is a   violation of HHS Guidelines and WCAG 2.0 Success Criterion 1.4.3. |
| RF-2 | Non-Informational   Images have alternative text | Images   used for decoration, layout, or other non-informative purposes have   alternative text. This issue is a violation of Section 508 and WCAG 2.0   Success Criterion 1.1.1. |

**Comment by aprematta** [comment]
Retest complete, issues have been fixed
